routes/portal/events.js

```diff
@@ -119,18 +121,20 @@ router.post('/meeting/update/:id', middleware.checkAuthenticated, function (
 	req,
 	res
 ) {
-	const name = req.body.meetingName;
-	const description = req.body.meetingDescription;
-	const link = req.body.meetingLink;
-	const time = req.body.meetingTime;
-	const day = req.body.meetingDay;
+	const sqlReq = new sql.Request();
+
+	sqlReq.input('name', sql.NVarChar, req.body.meetingName);
+	sqlReq.input('description', sql.NVarChar, req.body.meetingDescription);
+	sqlReq.input('link', sql.NVarChar, req.body.meetingLink);
+	sqlReq.input('time', sql.NVarChar, req.body.meetingTime);
+	sqlReq.input('day', sql.NVarChar, req.body.meetingDay);
 
-	var sqlQuery = `UPDATE tbl_meeting SET name = '${name}', description = '${description}', link = '${link}', time = '${time}', day = '${day}'
+	var sqlQuery = `UPDATE tbl_meeting SET name = @name, description = @description, link = @link, time = @time, day = @day
 				WHERE id=${req.params['id']}`;
```

Comment:
this is vulnerable to a sql injection

Reply:
to clarify its line 133, the where clause -- i saw you already fixed the prev line